src/404.html

```diff
@@ -115,7 +114,7 @@ <h2 class="section-heading">Newsletter</h2>
       </ul>
     </nav>
     <footer>
```

Comment:
I only converted 1 file index.html I can do all but thought i might leave some for someone else to do :D Currently you can convert one by one without affecting the build/deploy